pycode/epidemiology/epidata/geoModificationGermany.py

```diff
@@ -325,14 +340,14 @@ def merge_df_counties(df, merged_id, separated_ids, sorting=[dd.EngEng['date']],
         if not dd.EngEng['idCounty'] in columns:
             df_merged[dd.EngEng['idCounty']] = merged_id
         # check if column available
-        if dd.EngEng['county'] in df_merged.columns:
+        if dd.EngEng['county'] in df.columns:
```

Comment:
No. It shouldnt raise a key error if am right because it is an assignment. So if the column is not there, it will be recreated. It think there was an error in my first implementation. By merging, the county column may be removed and then the if does nothing (it is even superfluous then).